bson/registry.go

```diff
@@ -7,25 +7,12 @@
 package bson
 
 import (
-	"errors"
 	"fmt"
 	"reflect"
 	"sync"
 )
 
-// DefaultRegistry is the default Registry. It contains the default codecs and the
-// primitive codecs.
-var DefaultRegistry = NewRegistry()
```

Comment:
This change also covers the scope of [GODRIVER-3142](https://jira.mongodb.org/browse/GODRIVER-3142), which we should call out in the PR description.